src/bgw/job_stat.h

```diff
@@ -0,0 +1,29 @@
+#ifndef BGW_JOB_STAT_H
+#define BGW_JOB_STAT_H
+
+#include "catalog.h"
+#include "job.h"
+
+typedef struct BgwJobStat
+{
+	FormData_bgw_job_stat fd;
```

Comment:
Add `JobType` to this struct and set when you read the tuple.

Then we need not call `get_job_type_from_name()` all the time.

Reply:
fixed